openpype/tools/workfiles/app.py

```diff
@@ -673,7 +680,8 @@ def on_save_as_pressed(self):
             self._asset_id, self._task_name, self._task_type
         )
         create_workdir_extra_folders(
-            self.root,
+            self._workdir_path,
```

Comment:
Am I missing or reading over the lines where you changed the signature if this function? Because now you're adding an extra argument?

Reply:
At second glance it still looks like [this next line should be removed](https://github.com/pypeclub/OpenPype/pull/2504/files#diff-12cb7ca76388a7dccd9150f259570d5503ec1bc1bf77427406b20c0ed99b93c5R684). 

Or maybe you forgot to commit along the function change you did to [`openpype.lib.path_tools.create_workdir_extra_folders](https://github.com/pypeclub/OpenPype/blob/c94ce278f15428c4b717471378cb7f29c320988f/openpype/lib/path_tools.py#L205) but I don't think that needs different arguments, just passing only `self._workdir_path` should suffice.

Reply:
> At second glance it still looks like this next line should be removed.

Yes, didn't commit removement.